libc/src/__support/CPP/bit.h

```diff
@@ -25,6 +25,27 @@ namespace LIBC_NAMESPACE::cpp {
 #define LLVM_LIBC_HAS_BUILTIN_MEMCPY_INLINE
 #endif
 
+// Performs a copy from 'src' to 'dst' of 'size' bytes.
+// The semantics is valid if 'src' and 'dst' are equal but undefined if the
+// regions defined by [src, src + size] and [dst, dst + size] overlap.
+template <size_t size, typename DstT, typename SrcT>
+LIBC_INLINE constexpr void memcpy_inline(DstT *__restrict dst,
```

Comment:
is this function intended to be available for general internal use? If not, should it be in an `internal` namespace?

Reply:
We definitely need this function to implement the `memcpy` function. It was previously only defined in `libc/src/string/memory_utils/utils.h` but it's needed elsewhere now so I made it globally available in this header. It might not be the best place for it especially since it's not an official std function but I don't have a good home for it now. Do you have a suggestion? We can also leave it here, I don't think it hurts.

Reply:
With @legrosbuffle https://github.com/llvm/llvm-project/pull/75063#discussion_r1432534464 I think we can do away with this and put the function back into `libc/src/string/memory_utils/utils.h`.